neuro_data/static_images/data_schemas.py

```diff
@@ -944,7 +956,18 @@ def make(self, scan_key):
             tm[~valid] = -1
 
         self.insert1(dict(scan_key, treadmill=tm, valid=valid))
+    
 
+# Patch job for the hardcoding mess that was StaticMultiDataset.fill()
+# Instead of editing the code each time, the user will enter they scan with the desire group_id into here then call StaticMultiDataset.fill()
+@schema
+class StaticMultiDatasetGroupAssignment(dj.Manual):
+    definition = """
+    group_id : int unsigned
+    -> InputResponse
+    ---
+    description = '' : varchar(1024)
+    """
 
 @schema
 class StaticMultiDataset(dj.Manual):
```

Comment:
The purpose of this was to patch the stupid design of StaticMultiDataset while preserving its crap design since I was under the impression that a lot of code depends on it. Sure I can make it computed but problem is that it won't preserve the previous groups unless I insert them manually.

Reply:
wait, why?, yeah, all of the previous groups will be inserted manually (which you can in a Computed table) and new tables will be assigned the next highest id

Reply:
But what happens when people don't want to do that, like with the case where there were two scans assign to one id?

Reply:
oh yeah, you can do that . I guess Manual is ok. not sure what you made both group id and InputResponse part of the primary key but I guess it doesn't matter

Reply:
That has to do with allowing people to assign the a scan to multiple groups, I remember that there was old scan that had that problem hence why I had to make it like that.

Reply:
ok